src/Features/Core/Portable/AddImport/AbstractAddImportFeatureService.cs

```diff
@@ -223,6 +223,9 @@ private static bool IsHostOrRemoteWorkspace(Project project)
 
             foreach (var unreferencedProject in viableUnreferencedProjects)
             {
+                if (!unreferencedProject.SupportsCompilation)
```

Comment:
nit: feels like the check could be done in `GetViableUnreferencedProjects` instead, non-cs/vb projects should not be considered "viable"

Reply:
yeah, i did think about that.  for some reason i just felt it was clearer here.  like a "you shall not pass" sort of thing :)